.ci/deps.ghc-mod.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+# This causes ghc-mod to try to install them all
```

Comment:
expand note to indicate that building haskell deps from source is not working, with an issue